src/Frontend/Modules/Blog/Engine/Model.php

```diff
@@ -580,9 +580,9 @@ public static function getForTags(array $ids)
             'SELECT i.title, m.url
              FROM blog_posts AS i
              INNER JOIN meta AS m ON m.id = i.meta_id
-             WHERE i.status = ? AND i.hidden = ? AND i.id IN (' . implode(',', $ids) . ')
+             WHERE i.status = ? AND i.hidden = ? AND i.id IN (' . implode(',', $ids) . ') AND i.publish_on <= ?
              ORDER BY i.publish_on DESC',
-            array('active', 'N')
+            array('active', 'N', FrontendModel::getUTCDate('Y-m-d H:i') . ':00')
```

Comment:
Why append the seconds manually? Does SQL have improved performance if all times are rounded off to a minute?